codebundles/aws-c7n-network-health/sli.robot

```diff
@@ -0,0 +1,141 @@
+        ...    cmd=custodian run -r ${region} --output-dir ${OUTPUT_DIR}/${region}/aws-c7n-network-health ${CURDIR}/unused-eip.yaml --cache-period 0
+        ...    secret__aws_access_key_id=${AWS_ACCESS_KEY_ID}
+        ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+        ${count}=     RW.CLI.Run Cli
+        ...    cmd=cat ${OUTPUT_DIR}/${region}/aws-c7n-network-health/unused-eip/metadata.json | jq '.metrics[] | select(.MetricName == "ResourceCount") | .Value';
+        ${total_count}=    Evaluate    ${total_count} + int(${count.stdout})
+    END
+    ${unattached_eip_score}=    Evaluate    1 if ${total_count} <= int(${EVENT_THRESHOLD}) else 0
+    Set Global Variable    ${unattached_eip_score}
+
+Check for unused ELBs in AWS account `${AWS_ACCOUNT_ID}`
+    [Documentation]  Find unused Application Load Balancers (ALBs) and Network Load Balancers (NLBs) that do not have any associated targets
+    [Tags]    aws    elb    network 
+    ${total_count}=    Set Variable    0
+    FOR    ${region}    IN    @{AWS_ENABLED_REGIONS}
+        ${c7n_output}=    RW.CLI.Run Cli
+        ...    cmd=custodian run -r ${region} --output-dir ${OUTPUT_DIR}/${region}/aws-c7n-network-health ${CURDIR}/unused-elb.yaml --cache-period 0
+        ...    secret__aws_access_key_id=${AWS_ACCESS_KEY_ID}
+        ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+        ${count}=     RW.CLI.Run Cli
+        ...    cmd=cat ${OUTPUT_DIR}/${region}/aws-c7n-network-health/unused-elb/metadata.json | jq '.metrics[] | select(.MetricName == "ResourceCount") | .Value';
+        ${total_count}=    Evaluate    ${total_count} + int(${count.stdout})
+    END
+    ${unused_elb_score}=    Evaluate    1 if ${total_count} <= int(${EVENT_THRESHOLD}) else 0
+    Set Global Variable    ${unused_elb_score}
+
+Check for VPCs with Flow Logs disabled in AWS account `${AWS_ACCOUNT_ID}`
+    [Documentation]  Find VPCs that do not have Flow Logs enabled
+    [Tags]    aws    vpc    network 
+    CloudCustodian.Core.Generate Policy   
+    ...    ${CURDIR}/flow-log-disabled-vpc.j2    
+    ...    tags=${AWS_VPC_TAGS}
+    ${total_count}=    Set Variable    0
+    FOR    ${region}    IN    @{AWS_ENABLED_REGIONS}
+        ${c7n_output}=    RW.CLI.Run Cli
+        ...    cmd=custodian run -r ${region} --output-dir ${OUTPUT_DIR}/${region}/aws-c7n-network-health ${CURDIR}/flow-log-disabled-vpc.yaml --cache-period 0
+        ...    secret__aws_access_key_id=${AWS_ACCESS_KEY_ID}
+        ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+        ${count}=     RW.CLI.Run Cli
+        ...    cmd=cat ${OUTPUT_DIR}/${region}/aws-c7n-network-health/flow-log-disabled-vpc/metadata.json | jq '.metrics[] | select(.MetricName == "ResourceCount") | .Value';
+        ${total_count}=    Evaluate    ${total_count} + int(${count.stdout})
+    END
+    ${flow_log_disabled_vpc_score}=    Evaluate    1 if ${total_count} <= int(${EVENT_THRESHOLD}) else 0
```

Comment:
I think  ${EVENT_THRESHOLD}  needs a new variable name as it isn't properly descriptive. I believe you saw this in other code, but it was specifically referring to the amount of `events` that were related to a specific resource. In this specific case, the var should be something like ${DISABLED_FLOW_LOG_THRESHOLD}